python/cac_tripplanner/destinations/views.py

```diff
@@ -186,6 +186,25 @@ def get(self, request, *args, **kwargs):
             obj['point'] = json.loads(obj['point'].json)
             obj['image'] = image_to_url(obj, 'image')
             obj['wide_image'] = image_to_url(obj, 'wide_image')
+            # convert to format like properties on ESRI geocoder results
+            extent = {
+                'xmax': obj['point']['coordinates'][0],
+                'xmin': obj['point']['coordinates'][0],
+                'ymax': obj['point']['coordinates'][1],
+                'ymin': obj['point']['coordinates'][1]
+            }
+            obj['extent'] = extent
+            obj['attributes'] = {
+                'City': obj['city'],
+                'Postal': obj['zip'],
+                'Region': obj['state'],
+                'StAddr': obj['address']
+            }
+
+            obj['location'] = {
+                'x': obj['point']['coordinates'][0],
+                'y': obj['point']['coordinates'][1]
+            }
```

Comment:
Sure. Note that this is doing mostly the same object manipulations as the `SearchDestinations` view below it, but hadn't been updated as we were using `SearchDestinations` instead of this view. So should update both then.